package.json

```diff
@@ -20,17 +20,17 @@
     "test-update-snapshot": "jest -u",
     "cp-css": "./scripts/copy-css.sh",
     "clean-release": "rm -rf ./release; rm -rf ./dist",
-    "build": "npm run clean-release; rollup -c",
+    "build": "npm run rollup",
     "start": "rollup -c -w",
-    "reset": "rm -rf ./node_modules; npm install;",
+    "reset": "rm -rf ./node_modules; npm install",
     "prepublishOnly": "npm run build",
-    "predeploy": "cd example && npm install && npm run build",
+    "predeploy": "rm -rf ./release; rm -rf ./dist; npm install; npm run build",
     "storybook": "start-storybook -p 6006",
     "build-storybook": "build-storybook",
-    "eslint": "eslint --no-eslintrc  -c .eslintrc.js 'src/**'",
-    "stylelint": "stylelint 'src/**'",
     "lint": "npm run eslint; npm run eslint-ts",
+    "eslint": "eslint --no-eslintrc  -c .eslintrc.js 'src/**'",
     "eslint-ts": "eslint --no-eslintrc -c .eslintrc.json './src/**/*.ts*'",
+    "stylelint": "stylelint 'src/**'",
     "generate-component": "plop",
     "rollup": "npm run clean-release; rollup -c",
```

Comment:
just remove this and keep build as it was

Reply:
ok ok